FragmentsAssignment/FragmentsAssignment/app/src/main/res/layout/fragment_home_screen.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
Should have used Relative Layout
Added button as first element with property alignParentBottom=true
And listView as above button and alignParentTop